sync/class.jetpack-sync-module-comments.php

```diff
@@ -40,6 +43,30 @@ public function init_listeners( $callable ) {
 		$this->init_meta_whitelist_handler( 'comment', array( $this, 'filter_meta' ) );
 	}
 
+	public function handle_comment_contents_modification( $new_comment, $old_comment, $new_comment_with_slashes ) {
+		$content_fields = array(
+			'comment_author',
+			'comment_author_email',
+			'comment_author_url',
+			'comment_content',
+		);
+		$changes = array();
+		foreach ( $content_fields as $field ) {
+			if ( $new_comment_with_slashes[$field] != $old_comment[$field] ) {
+				$changes[$field] = array( $new_comment[$field], $old_comment[$field] );
+			}
+		}
+
+		if ( ! empty( $changes ) ) {
+			$action_args = array(
+				$new_comment['comment_ID'],
+				$changes,
+			);
+			do_action( 'jetpack_modified_comment_contents', $action_args );
```

Comment:
Lets call the do action like this 
`do_action( 'jetpack_modified_comment_contents',$new_comment['comment_ID'], $changes );` 
This adds some a bit more consistency